doc/api/tracing.md

```diff
@@ -47,4 +46,35 @@ as the one used by `process.hrtime()`
 however the trace-event timestamps are expressed in microseconds,
 unlike `process.hrtime()` which returns nanoseconds.
 
+## The `trace_events` module
+
+The `trace_events` module may be used to turn trace event reporting on and off.
+
+It is accessible using `require('trace_events')`.
+
+### trace_events.setTracingCategories(categories)
```

Comment:
I'm still in favor of an API that allows one to enable and disable categories individually, as that will make it easier to have different modules enable categories.

Reply:
Yes, I was going back and forth on that. It wouldn't be too difficult to do I just wasn't sure just how far to go down that road so opted for the easiest thing initially.

What I had in mind was a simple variation on this such as:

```js
// trace.events.enableTracingCategories(...categories)
// trace.events.disableTracingCategories(...categories)
trace_events.enableTracingCategories('node', 'v8', 'foo')
trace_events.disableTracingCategories('node')

trace_events.getTracingCategories() // Returns an array, e,g, ['v8', 'foo']
```

Reply:
Yes, it was something like that I had in mind.